src/platforms/common/configuration/options.mdx

```diff
@@ -497,13 +497,13 @@ The callback typically gets a second argument (called a "hint") which contains t
 
 </ConfigKey>
 
-<PlatformSection supported={["javascript", "java", "python", "node", "php", "dotnet", "flutter"]}>
+<PlatformSection supported={["javascript", "java", "python", "node", "php", "dotnet", "dart"]}>
```

Comment:
Ignore those changes, I just fixed a bug while checking out the preview.

Reply:
Do you mean ignore all the updates where you've updated the supported list for a platform section? Or just the one above?

Reply:
All the updates, the only important one is https://github.com/getsentry/sentry-docs/pull/5330#discussion_r928810993 that you already reviewed, thanks @imatwawana